src/octree/bytecode.rs

```diff
@@ -115,7 +115,7 @@ where
                 if !is_leaf {
                     let count;
                     match list.next_object()?.unwrap() {
-                        Object::Integer(i) => count = i.parse::<u32>().ok().unwrap(),
+                        Object::Integer(i) => count = i.parse::<u8>().ok().unwrap(),
```

Comment:
This change should not be here, but it was required to have this after the latest change on main caused a conflict